charts/celeborn/values.yaml

```diff
@@ -36,10 +36,26 @@ workerReplicas: 5
 securityContext:
   fsGroup: 10006
 
+# Current Celeborn support followings volume type:
+# - emptyDir
+# - hostPath
+# Note: sizeLimit only works in emptyDir type
+volume:
+  master:
+    ratis:
+      path: &master-volume-ratis /mnt/rss_ratis
+      type: hostPath
+      sizeLimit: 1Gi
+  worker:
+    dirs:
+      path: &worker-volume-dirs /mnt/disk1,/mnt/disk2,/mnt/disk3,/mnt/disk4
+      type: hostPath
+      sizeLimit: 1Gi
+
 # celeborn configurations
 celeborn:
-  celeborn.ha.master.ratis.raft.server.storage.dir: /mnt/rss_ratis/
-  celeborn.worker.storage.dirs: /mnt/disk1,/mnt/disk2,/mnt/disk3,/mnt/disk4
+  celeborn.ha.master.ratis.raft.server.storage.dir: *master-volume-ratis
```

Comment:
please use go template syntax, mixed-using yaml/go template var ref syntax for the same purpose increases the complication